test/data/event.create.full.yaml

```diff
@@ -0,0 +1,9 @@
+# Event Create Example
+sha: 46f1a0bd5592a2f9244ca321b129902a06b53e03
```

Comment:
Is `sha` a required value when creating event ? If it's empty will it default to HEAD ?

Reply:
I don't think it's needed from the API. We can look it up using getCommitSha